scripts/zones/Buburimu_Peninsula/mobs/Wake_Warder_Wanda.lua

```diff
@@ -0,0 +1,48 @@
+-----------------------------------
+-- Area: Buburimi Peninsula
+--  MOB: Wake Warder Wanda
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_MAGIC_COOL, 50); -- just one spell to spam
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:addMod(MOD_SILENCERES,20);
+    mob:addMod(MOD_BINDRES,20);
+end;
+
+-----------------------------------
+-- onMobEngaged
+-----------------------------------
+
+function onMobEngaged(mob,target)
+    mob:setMod(MOD_REGAIN, 25);
+end;
+
+-----------------------------------
+-- onMobDisengage
+-----------------------------------
+
+function onMobDisengage(mob)
+    mob:setMod(MOD_REGAIN,0);
+end;
+
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob, player, isKiller)
+    UpdateNMSpawnPoint(mob:getID());
+    mob:setRespawnTime(math.random(3600,4200)); -- repop 60-70min
```

Comment:
probably dont want this running for every player in the party, better to have it run once (isKiller)